craft_store/store_client.py

```diff
@@ -139,6 +143,7 @@ def login(
         ttl: int,
         packages: Optional[Sequence[endpoints.Package]] = None,
         channels: Optional[Sequence[str]] = None,
+        save_credentials=True,
```

Comment:
Is there a case where login should be called when using auth from an env var?

Reply:
I don't understand if you are asking for a change or asking a question. I would say yes for this library and no for Charmcraft specifically.